scripts/cmake/conan/conan.cmake

```diff
@@ -81,6 +81,11 @@ function(conan_cmake_settings result)
         list(GET VERSION_LIST 1 MINOR)
         conan_cmake_detect_gnu_libcxx(_LIBCXX)
         set(_SETTINGS ${_SETTINGS} -s compiler=gcc -s compiler.version=${MAJOR}.${MINOR} -s compiler.libcxx=${_LIBCXX})
+    elseif (${CMAKE_CXX_COMPILER_ID} STREQUAL Intel)
```

Comment:
Why? Is it, because conan does not provide pre-built libs for icc?

Reply:
Ups sorry wrong branch!